js/iiif-iabookreader_strawberry.js

```diff
@@ -34,7 +34,7 @@
                         var options = {
                             ui: 'full', // embed, full (responsive)
                             el: '#' + element_id,
-                            iiifmanifesturl: 'https://iiif.archivelab.org/iiif/platowithenglish04platuoft/manifest.json',
+                            iiifmanifesturl: drupalSettings.format_strawberryfield.iabookreader[element_id]['manifesturl'],
```

Comment:
Hi, i would recommend testing if the value exists and is not empty, just in case. Just in case, we should really trust that our settings are well formed and to be honest if i ask you to do that i will need to do the same for the whole structure =)

Reply:
I added URL check into StrawberryPagedFormatter.php, I think checks better before (in PHP) than here, do you agree @DiegoPino ?

Reply:
Yes. So let me understand. If formatter passes a manifest and there is no inline Manifest, URL will work right? If the full manifest is passed, URL is not used. Right? I know its my code but i need a double pair of eyes!

Reply:
Yes, exactly as you wrote the code:
if (self.options.iiifmanifest != null) {
...
else {
// Ok, no full manifest, now try with the remote URL
...

Reply:
Perfect.. did i write that? jajja. Thanks for refreshing my mind.